src/download.ts

```diff
@@ -250,18 +249,8 @@ export async function getTranslators(targetLanguageIds: string[]): Promise<strin
  *
  * @returns The build id.
  */
-export async function buildProject(): Promise<number> {
-	if (process.env.CROWDIN_ORG) {
-		const { id, status } = (await translationsApi.listProjectBuilds(PROJECT_ID, undefined, 1)).data[0].data;
-		if (status === 'finished') {
-			return id;
-		}
-	}
-	const { id, status } = (
-		await translationsApi.buildProject(PROJECT_ID, {
-			skipUntranslatedStrings: true
-		})
-	).data;
+export async function buildTranslations(): Promise<number> {
+	const { id, status } = (await translationsApi.buildProjectDirectoryTranslation(PROJECT_ID, 738, { skipUntranslatedStrings: true })).data;
```

Comment:
Please describe/define this `738` - right now it just looks like a magic number.